core/src/test/scala/unit/kafka/admin/LeaderElectionCommandTest.scala

```diff
@@ -273,7 +277,7 @@ final class LeaderElectionCommandTest extends ZooKeeperTestHarness {
       LeaderElectionCommand.main(
         Array(
           "--bootstrap-server", bootstrapServers(servers),
-          "--election-type", "preferrred"
+          "--election-type", "preferred"
```

Comment:
Hmm. This test worked even with the misspelling?

Reply:
My guess is validation for election type happens after validation for required arguments.